### ISSUE TITLE
Containers won't run without sudo on Ubuntu 24.04

### ISSUE THREAD

**Issue by gerardcanal** [apptainer/apptainer#2457]
### Version of Apptainer
1.3.3

### Expected behavior

The container runs without sudo.

### Actual behavior

There is an error when runnning the container without sudo:

```
ERROR  : Could not write info to setgroups: Permission denied
ERROR  : Error while waiting event for user namespace mappings: no event received
```
The container seems to work well when run with sudo.
This happens on previously working containers after a fresh install of Ubuntu 24.04. Unsure if related to https://github.com/apptainer/apptainer/issues/2027 but I understand 1.3.3 fixes the apparmor issues.

### Steps to reproduce this behavior
Running a container that was working well in Ubuntu 20.04.

### What OS/distro are you running

Ubuntu 24.04 LTS

### How did you install Apptainer

Using the .deb from Github, version 1.3.3

**Comment by DrDaveD** [comment]
That is an unexpected error message, but Apptainer 1.3.3 is known to not work unprivileged in Ubuntu 24.04 because of an unfortunate decision of the Ubuntu maintainers to restrict user namespaces in combination with a bug in Apptainer 1.3.3.  See instructions in [INSTALL.md](https://github.com/apptainer/apptainer/blob/release-1.3/INSTALL.md#apparmor-profile-ubuntu-2310) of what to do about it for now.  #2364 should fix it for Apptainer 1.3.4.

I think the specific error message must have something to do with either the container you're using or the combination of options you're using to start it.

**Comment by gerardcanal** [comment]
Hi @DrDaveD!
Thanks, I missed that. This worked well. The error happens even without any option, and a vanilla apptainer install. All the containers I tried come from the same dockerhub base one, so it may be the container.
